### DISCUSSION TITLE
How do I apply this to photos of paper?

### DISCUSSION THREAD

**Discussion by hrehfeld** [sbrunner/scan-to-paperless#812] [General]
I'd like to try and use this with my cellphone camera, is there a good way to use this on just a folder of images?

**Comment by sbrunner** [comment]
It's not the base use case, the use case it to use the scan command, and she will use a scanner to create the images with a configuration file.
If you want to do a try, you can create a configuration file name (named `config.yaml`) like that:
```
images:
  - <image 1 filename>
  - ...
```
Then you can try to run `docker run --volume=$(pwd):/source --volume=$(pwd)/destination:/destination sbrunner/scan-to-paperless`
But I never do that, then I don't know whether the result will be good or not...

**Comment by hrehfeld** [comment]
Thank you, gonna try when I have time!
